client/src/app/_services/auth.service.ts

```diff
@@ -26,40 +26,30 @@ export class AuthService {
       if (currentUser.jwt) {
         const jwtDecoded = jwt_decode(currentUser.jwt);
         if (Date.now() < jwtDecoded.exp * 1000) {
```

Comment:
We should probably check also if the token is soon to be expired. If yes, we should disconnect the user  directly to prevent disconnecting him during his work

Reply:
Great idea 👍 But what is 'soon' ?

Reply:
Ok so let's say that it's 12h. If you disagree, tell me :)